adminapp/src/pages/VendorAccountDetailPage.jsx

```diff
@@ -0,0 +1,130 @@
+import api from "../api";
+import AdminLink from "../components/AdminLink";
+import BoolCheckmark from "../components/BoolCheckmark";
+import DetailGrid from "../components/DetailGrid";
+import RelatedList from "../components/RelatedList";
+import ResourceDetail from "../components/ResourceDetail";
+import { dayjs } from "../modules/dayConfig";
+import SafeExternalLink from "../shared/react/SafeExternalLink";
+import React from "react";
+
+export default function VendorAccountDetailPage() {
+  return (
+    <ResourceDetail
+      apiGet={api.getVendorAccount}
+      title={(model) => `Vendor Account ${model.id}`}
+      properties={(model) => [
+        { label: "ID", value: model.id },
+        { label: "Created At", value: dayjs(model.createdAt) },
```

Comment:
I think these all need `.format`, are we doing this elsewhere? It'd use the default dayjs formatting which may not be what we want (let's be consistent with time formatting).